src/screens/WebScreen.js

```diff
@@ -3,17 +3,23 @@ import React, { useContext } from 'react';
 import { ActivityIndicator, StyleSheet, TouchableOpacity, View } from 'react-native';
 import { WebView } from 'react-native-webview';
 
-import { colors, normalize } from '../config';
+import { colors, consts, normalize } from '../config';
 import { Icon, LoadingContainer, SafeAreaViewFlex, WrapperWithOrientation } from '../components';
 import { arrowLeft } from '../icons';
 import { OrientationContext } from '../OrientationProvider';
+import { useMatomoTrackScreenView } from '../hooks';
 
 export const WebScreen = ({ navigation }) => {
   const { orientation, dimensions } = useContext(OrientationContext);
   const webUrl = navigation.getParam('webUrl', '');
+  const { MATOMO_TRACKING } = consts;
 
   if (!webUrl) return null;
 
+  // NOTE: if we are able to navigate to a web screen from another web screen, we would need an own
+  //       useEffect here for tracking the screen view with the `webUrl` as dependency
+  useMatomoTrackScreenView(`${MATOMO_TRACKING.SCREEN_VIEW.WEB} / ${webUrl}`);
```

Comment:
this hook is after an early return in line 17.
we should handle this differently.
maybe use a `useEffect` here with a condition regarding `webUrl` inside

Reply:
yes, this would also tackle a possible future case mentioned already in the comment.

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/109/commits/6ceb63d42d8d4ba4e0540541023b81c2f26a3e11